Services/FacebookOAuth/FacebookProfileManager.cs

```diff
@@ -62,7 +62,7 @@ private async Task<UserProfile> GetProfileFromAccessTokenAsync(FacebookTokensMod
             string fields = ReformScopeToFields.Transform(tokensModel.Scope);
             
             string url = facebookProperties.GetProfileLink + tokensModel.AccessToken +
-                    "&fields=first_name,last_name,name,gender,location" + fields;
+                    "&fields=first_name,last_name,name,gender,location,picture" + fields;
```

Comment:
Is it only for test?

Reply:
It's default fields for getting main information about user from Facebook